02- Running ADO Scanner from command line/Readme.md

```diff
@@ -122,6 +124,22 @@ The Get-AzSKADOSecurityStatus command now supports checkpointing via a "-UsePart
 Get-AzSKADOSecurityStatus-OrganizationName "<OrganizationName>" -ScanAllResources -UsePartialCommits
 ```
 
+#### Speed up checkpointed scans with "-DoNotRefetchResources" switch
+The "-UsePartialCommits" switch also supports an optional switch: "-DoNotRefetchResources" in SDL mode. When this switch is used, resources are not re-fetched during the continuation of the checkpointed scan (i.e., when the "-upc" switch is used). This efficiently speeds up scans of subsequent batches after the initial one. Currently the resources supported with the switch are: Release, Agent Pool, Organization and Project. 
+
+```PowerShell
+Get-AzSKADOSecurityStatus-OrganizationName "<OrganizationName>" -ProjectName "<ProjectName>" -ReleaseNames * -ResourceTypeName -Release -UsePartialCommits -DoNotRefetchResources
```

Comment:
add space between "Get-AzSKADOSecurityStatus"  & "-OrganizationName" in all commands